pandas/core/series.py

```diff
@@ -556,7 +552,8 @@ def _init_dict(
     # ----------------------------------------------------------------------
 
     @property
-    def _constructor(self) -> Callable[..., Series]:
+    # error: Cannot override writeable attribute with read-only property
+    def _constructor(self) -> Callable[..., Series]:  # type: ignore[override]
```

Comment:
Could potentially be `def _constructor(self: _SeriesT) -> type[_SeriesT]` (or `def _constructor(self) -> type[Self]`). The same could also be done for `PandasObject._constructor`

Reply:
The trouble with that is _constructor can return something other than a class, xref #51772 id like to change that